package.json

```diff
@@ -1,17 +1,24 @@
 {
-    "name": "react-focus-rings-base",
-    "private": true,
-    "version": "1.0.0",
-    "description": "",
-    "main": "index.js",
-    "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "keywords": [],
-    "author": "Jon",
-    "license": "ISC",
-    "devDependencies": {
-      "@types/node": "^18.7.16",
-      "typescript": "^4.8.3"
-    }
-  }
\ No newline at end of file
+  "name": "react-focus-rings-base",
+  "private": true,
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "fix": "eslint --fix '**/*.tsx'",
+    "lint": "eslint"
+  },
+  "keywords": [],
+  "author": "Jon",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/node": "^18.7.16",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.14.0",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.0.1",
```

Comment:
as a nit, i'd prefer to keep eslint and prettier separate. the formatting rules in ESLint have been deprecated and unrecommended in 8 and will be removed in 9, so there's no longer a need to make sure they don't conflict, and we can just have a separate `format` script to run prettier, then combine them in a `check` or `ci` script to run both on CI (separate jobs isn't useful here, since the project is so small).